src/main/java/seedu/contax/model/appointment/Appointment.java

```diff
@@ -65,6 +90,21 @@ public Person getPerson() {
         return person;
     }
 
+    public Priority getPriority() {
+        return priority;
+    }
+
+    public String getPriorityToString() {
+        if (priority != null) {
+            return priority.toString();
+        }
+        return "NONE";
+    }
```

Comment:
What is the point of this method? You can just use `getPriority()`

Reply:
You can use `Object.equals` in the appointment equals check.
It is also possible to override an enum's toString method.

Reply:
Ok thx for suggestion, I changed